File: It is My Serve.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int p,q;
        scanf("%d%d",&p,&q);
        if((p+q)%4==0||(p+q)%4==1)
        printf("Alice\n");
        else
        printf("Bob\n");
    }
}

File: It is My Serve.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            if((a+b)%4==0||(a+b)%4==1)
            System.out.println("Alice");
            else
            System.out.println("Bob");
        }
    }
}